app/api/route/handler/example_person_handler.py

```diff
@@ -0,0 +1,58 @@
+from datetime import date
+from typing import Optional
+from uuid import uuid4
+
+from pydantic import UUID4
+
+import api.logging
+from api.db.models.example_models import ExamplePerson, ExamplePet
+from api.util.api_context import ApiContext
+from api.util.pydantic_util import PydanticBaseModel
```

Comment:
the problem with "util" modules is that they become an overly centralized dependency. this is a situation where DRY is actually harmful. instead, we should have something like `class BaseRequestModel` which we define in the `app.api.route` module or `app.api.handler` module, and if we use Pydantic in another context that isn't request handling (for example if our internal application data models layer also uses Pydantic), we should define a separate base class for that use case. The various contexts of the application should hide the underlying libraries from each other.

relatedly, in the context of request handling, I don't think we need to set `orm_mode = true`, since request_body is always a dictionary.

Reply:
I could restructure the files a bit. Ended up with more in the util folder than I'd like. Some truly generic pieces like the string & file utils might make sense to keep there, but you're right that api-specific components should probably go.

I believe `orm_mode = True` is for the reverse. Being able to convert the SqlAlchemy object into the response object.

Reply:
> I believe orm_mode = True is for the reverse. Being able to convert the SqlAlchemy object into the response object.

yeah that makes sense, i guess we can defer this conversation until the future where we resolve the other [discussion on doing pydantic model validation on the server response](https://github.com/navapbc/wic-mt-demo-project-mock-api/pull/20#discussion_r970143239)